reference/calendar.md

```diff
@@ -1,31 +1,27 @@
 (team/calendar)=
-# Team Calendars
+# Team calendars
 
+(calendar:events)=
 ## 2i2c Events
 
 Below is a calendar with some relevant events for the 2i2c Team and Community.
 You can find a [link to the team calendar here](https://calendar.google.com/calendar/u/1?cid=Y180aGpqb3VvamQ4cHNxbDlpMWE4bmQxdWZmNEBncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
-```{dropdown} Team events calendar
+```{dropdown} Team Events calendar (times in UTC)
 
-<iframe src="https://calendar.google.com/calendar/embed?height=600&amp;wkst=1&amp;bgcolor=%23ffffff&amp;ctz=America%2FLos_Angeles&amp;src=Y180aGpqb3VvamQ4cHNxbDlpMWE4bmQxdWZmNEBncm91cC5jYWxlbmRhci5nb29nbGUuY29t&amp;src=Y19pNTJqZGNhbTZ0M3FsaDF1NTNqdG42MjNwY0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t&amp;src=Y184ZmhrOXBtZmxocWM3OWI2bWY0dnEwYjlwc0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t&amp;color=%23009688&amp;color=%23616161&amp;color=%234285F4&amp;showTitle=0&amp;showPrint=0&amp;showTabs=0&amp;showCalendars=1" style="border:solid 1px #777" width="800" height="600" frameborder="0" scrolling="no"></iframe>
+<iframe src="https://calendar.google.com/calendar/embed?src=c_4hjjouojd8psql9i1a8nd1uff4%40group.calendar.google.com" style="border: 0; width:100%"  height="600" frameborder="0" scrolling="no"></iframe>
 
 ```
 
 (team-roles-calendar)=
 ## Team Roles Calendar
 
 We use a Team Roles calendar to keep track of when each team member is serving in a given role.
-You can find [the Team Roles calendar here](https://calendar.google.com/calendar/embed?src=c_nq8hl7qsm484g1p7mfkm29jpo8%40group.calendar.google.com&ctz=America%2FLos_Angeles).
+You can find [the Team Roles calendar here](https://calendar.google.com/calendar/u/1?cid=Y19ucThobDdxc200ODRnMXA3bWZrbTI5anBvOEBncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
```

Comment:
Another `/u/1` link here

Reply:
gah this drives me nuts - thanks for finding